activities/LabyrinthJS.activity/lib/sugar-web/activity/activity.js

```diff
@@ -58,6 +58,8 @@ define(["webL10n",
         var activityPalette = new activitypalette.ActivityPalette(
             activityButton, datastoreObject);
 
+        document.getElementById("activity-palette").style.zIndex = "3";
```

Comment:
You must not change code in sugar-web library. The code for this library must be the same for all activities.

Reply:
Okay sorry @llaske. 
I will try to solve this bug via styling changes in .css files.